bundles/org.openhab.binding.gpio/README.md

```diff
@@ -78,7 +81,7 @@ Thing gpio:pigpio-remote:sample-pi-1 "Sample-Pi 1" [host="192.168.2.36", port=88
 Thing gpio:pigpio-remote:sample-pi-2 "Sample-Pi 2" [host="192.168.2.37", port=8888] {
     Channels:
         Type pigpio-digital-input : sample-input-3 [ gpioId=16, debouncingTime=20]
-        Type pigpio-digital-input : sample-input-4 [ gpioId=17, invert=true, debouncingTime=5]
+        Type pigpio-digital-input : sample-input-4 [ gpioId=17, invert=true, debouncingTime=5, pullupdown=2]
```

Comment:
It feels like a very direct way, would it make sense to expose a bit less direct, or not. I'm in doubt.

Reply:
Have you got any suggestion for improvement ?

Reply:
Do you mean like `pullupdown=OFF`, `pullupdown=UP`, `pullupdown=DOWN`? Would be better from the usability perspective, IMHO. But I won't insist on changing it.

Reply:
Let's merge it then

Reply:
OK, @SloCompTech feel free to file a follow-up PR if you agree on these thoughts.

Reply:
Ok, will do, so I need to change parameter type to string or is there another more suitable type like a constant ?

Reply:
There's no enum type or so. You need to make it `text`. See https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions for the available types.